domain-ee/ee-max-cfi-app/scripts/dc_lookup_table_updater.py

```diff
@@ -45,11 +46,23 @@ def import_file(filename: str) -> dict[int, tuple[Any, Any, Any, Any, Any, Any]]
             except ValueError:
                 raise ValueError(f'Invalid diagnostic code at index {index}: \n{csv_line}')
 
+            rated_issue_name = remove_diagnostic_code_prefix(clean_string(rated_issue_name), diagnostic_code_str)
             diagnostic_code_to_data[diagnostic_code] = (rated_issue_name, max_rating_str, body_system, category, subcategory, cfr_ref)
 
     return diagnostic_code_to_data
 
 
+def clean_string(input_string):
+    # Remove non-alphanumeric characters, excluding some special characters, from the end of the string
+    return re.sub(r'[^\w(){}\[\]°]*$', '', input_string)
```

Comment:
Neat, have we seen a condition name end with a degree sign before?

Reply:
Yep! There's 5208: "Forearm, flexion limited to 100° and extension to 45°"